test/test_fx.py

```diff
@@ -468,7 +468,7 @@ def lower_to_elementwise_interpreter(orig_mod : torch.nn.Module) -> torch.nn.Mod
                             # Pull out constants. These constants will later be
                             # fed to the interpreter C++ object via add_constant()
                             arg_name = f'constant_{constant_idx}'
-                            constants[arg_name] = torch.Tensor(
+                            constants[arg_name] = torch.tensor(
```

Comment:
Since we are pulling out constants, I guess we don't have to worry about their types, right?

Reply:
this looks fine to me